### PR TITLE
Updated User guide Getting Started

### PR DESCRIPTION
**Pull request by shaowi**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -66,82 +57,60 @@ Unlike a traditional to-do list app, Le Tracker is tailored to meet these needs
...
+### Tutorials and Examples
 
-### Edit
+Scenario 1:
 
-- `edit {module_code} [/code {updated_code}] [/name {updated_name}] [/tags {tag_1}, [{tag_2}, ...]]`: Edits the details of a module in Le Tracker
-- `edit {lecture_name} [/mod {module_code}] [/name {updated_name}] [/tags {tag_1}, [{tag_2}, ...]]`: Edits the details of a lecture
-- `edit {video_name} [/mod {module_code}] [/lec {lecture_name}] [/name {updated_name}] [/timestamp {updated_timestamp}] [/watch] [/unwatch] [/tags {tag_1}, [{tag_2}, ...]]`: Edits the details of a video
+1. To add a module, run `add CS2103 /name Software Engineering`.
+1. To give it a tag of `BestModule`, run `tag CS2103 /tags BestModule`.
```
[the remark this answers is not in the document]
Comment (lennoxtr) [reply docs/UserGuide.md:85]: If the intended meaning is that the commands are unrelated to each other, I think using a hyphen is better than a number. Like,

- To add a module...
- To give it a tag of ...

[the remark this answers is not in the document]
Comment (shaowi) [reply docs/UserGuide.md:85]: For markdown, this wouldn't matter cuz what you previewed will show the numbering in ordered sequence

[the remark this answers is not in the document]
Comment (lennoxtr) [reply docs/UserGuide.md:85]: Oh whoops I see it now

**Review by lennoxtr — APPROVED** [review]
LGTM

**Pull request merged**